src/julia_internal.h

```diff
@@ -1016,6 +1016,10 @@ void jl_log(int level, jl_value_t *module, jl_value_t *group, jl_value_t *id,
 
 int isabspath(const char *in);
 
+// TODO(NHDALY): Find the right spot for this.
+JL_DLLEXPORT void jl_resolve_globals_in_ir(jl_array_t *stmts, jl_module_t *m,
+                              jl_svec_t *sparam_vals, int binding_effects);
```

Comment:
this should go in `julia.h` if you want to depend on it. also take it out of `src/toplevel.c`